src/less/dark-mode/base/dark-mode.less

```diff
@@ -27,6 +27,7 @@
         --color-status-general: @color-status-general-dark-mode;
         --color-separator: @color-separator-dark-mode;
         --color-image-border: @color-image-border-dark-mode;
+        --checkbox-mixed-foreground: @color-black;
```

Comment:
Should be `--color-checkbox-mixed-foreground` to follow the established naming convention.